src/rdo.rs

```diff
@@ -766,6 +766,17 @@ pub fn rdo_tx_size_type<T: Pixel>(
   (best_tx_size, best_tx_type)
 }
 
+#[inline]
+fn mv_in_range(mv: MotionVector, ref_mv: MotionVector) -> bool {
+  const MAX_MV_MAG: i32 = 1 << 14;
```

Comment:
so this constant is for 1/8 pixel, Q3, right? So, in integer, 2048 pixel positions.

Reply:
I identified `context::MV_UPP` as the name for this constant.